stable/yugaware/templates/init-container-script.yaml

```
@@ -0,0 +1,49 @@
+{{- if .Values.securityContext.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}-yugaware-init-container
+  labels:
+    app: {{ template "yugaware.name" . }}
+    chart: {{ template "yugaware.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Values.helm2Legacy | ternary "Tiller" (.Release.Service | quote) }}
+data:
+  init-permissions.sh: |
+    #!/bin/bash
+
+    set -xe
+
+    directory_name="/opt/yugabyte/yugaware/data"
+    pemfiles=$(find "${directory_name}/keys/" -name "*.pem" -exec stat -c "%a" {} + | uniq | tr '\n' ',')
+    IFS="," read -r -a array <<< "$pemfiles"
```

```suggestion
    IFS="," read -r -a pemfile_perms <<< "$pemfiles"
```
Couldn't think of anything better, but should be readable than just array.